webapp/src/client.ts

```diff
@@ -112,8 +112,7 @@ export default class CallsClient extends EventEmitter {
         this.ws = ws;
 
         ws.on('error', (err) => {
-            console.log(`ws error: ${err}`);
-            this.ws = null;
```

Comment:
This was preventing the ws connection to disconnect in case of server side error (e.g. during join).

Reply:
Nice catch!